### PR TITLE
Add the ability to display remaining prayer time in orb

### PR DESCRIPTION
**Pull request by seandewar**
Adds a configuration option for showing the remaining prayer time next to the orb in seconds:

image

If the user is not praying, the percentage will be shown instead as normal.

Similar to the feature proposed in PR #4804 for run energy.

### CODE REVIEW

**Comment by MagicfTail** [comment]
This is already a feature? https://i.imgur.com/vIfXzkG.png

**Comment by MagicfTail** [comment]
Nvm, i see

File: runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java [hunk 1]
```diff
@@ -265,6 +306,53 @@ private boolean isAnyPrayerActive()
 		return false;
 	}
 
+	double getPrayerDrainRate(Client client)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java]: This can be private

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java]: Also remove Client from parameters

**Comment by dekvall** [comment]
Added in #9599

**Pull request closed without merging**